### ISSUE TITLE
Update agency reporting page as the info we require has changed.

### ISSUE THREAD

**Issue by agautsc** [feed-and-report/main#9]

**Comment by agautsc** [comment]
@Kplow76 can you go into more detail on these updates? I'm going to need to sketch out the exact fields to give a proper quote.

**Comment by agautsc** [comment]
@Kplow76 there is also a 'Combo report' that is created where you can see the report numbers the agency submits and the food poundage by category donated to that agency during that time. 

Do you still use that report and find it valuable?

**Comment by Kplow76** [comment]
Yes...I still use the combo report.  I need to be able to compile the information from the agency reports for United Way Reporting.

For now, the fields can stay the same for the Agencies.  It will be easier for me to update the hard copy for those that use it and make it match the online version.

Thanks,
Katie
